### DISCUSSION TITLE
The write is not working

### DISCUSSION THREAD

**Discussion by sauli98** [GorvGoyl/ChatGPT-Writer#62] [Report Bugs and Issues]
I passed the Cloudflare once i click the link  but there is no response after I click the generate email button

**Comment by gubinzhousong** [comment]
ChatGPT Writer is not working. I removed and added hundred times, still not working. It says "Please reload the website you are on and then use the extension. Make sure you're on a regular website like Gmail, not browser settings or Chrome extensions store (chrome.google.com)". No matter what I try, it is just not working.
